src/test/docker/jammy/Dockerfile

```diff
@@ -30,6 +30,12 @@ RUN apt-get update \
         jq \
  && rm -rf /var/lib/apt/lists/*
 
+# ubuntu containers now "minimize" themselves, so manpages aren't installed
+# to warn people about this, /usr/bin/man is a shell script that doesn't bother
```

Comment:
nit: here and in commit message.  I think period after "installed" and upper case "to"?   Took a few reads to figure out.

Reply:
```suggestion
# Ubuntu containers now "minimize" themselves, so manpages aren't installed.
# To warn people about this, /usr/bin/man is a shell script that doesn't bother
```